malariagen_data/ag3.py

```diff
@@ -341,3 +347,100 @@ def _results_cache_add_analysis_params(self, params):
         super()._results_cache_add_analysis_params(params)
         # override parent class to add AIM analysis
         params["aim_analysis"] = self._aim_analysis
+
+    def load_inversion_tags(self, inversion: str):
+        # needs to be modified depending on where we are hosting
+        path = "malariagen_data/resources/karyotype_tag_snps.csv"
+        df_tag_snps = pd.read_csv(path, sep=",")
+        return df_tag_snps.query(f"inversion == '{inversion}'").reset_index()
+
+    def karyotype_tags_n_alt(self, gt, alts, inversion_alts):
```

Comment:
This looks like it could be a function rather than a method, because `self` is never accessed. Suggest to move this to a standalone function. Also suggest to rename as `_karyotype_tags_n_alt` to indicate it's private, i.e., not intended for public API.